coderd/users.go

```diff
@@ -1316,6 +1323,25 @@ func (api *API) CreateUser(ctx context.Context, store database.Store, req Create
 	return user, req.OrganizationID, err
 }
 
+// findUserAdmins fetches all users with user admin permission including owners.
+func findUserAdmins(ctx context.Context, store database.Store) ([]database.GetUsersRow, error) {
+	// Notice: we can't scrape the user information in parallel as pq
+	// fails with: unexpected describe rows response: 'D'
+	owners, err := store.GetUsers(ctx, database.GetUsersParams{
+		RbacRole: []string{codersdk.RoleOwner},
+	})
+	if err != nil {
+		return nil, xerrors.Errorf("get owners: %w", err)
+	}
+	userAdmins, err := store.GetUsers(ctx, database.GetUsersParams{
```

Comment:
Do we also need to include org user admins here?

Reply:
Probably, I will leave it as a follow-up once the org part is final. Most likely we need to adjust _create account_ logic too.